lms/security.py

```diff
@@ -122,6 +126,11 @@ def get_policy(path: str):
                 partial(get_lti_user_from_bearer_token, location="form")
             )
 
+        if path in {"/email/settings", "/email/unsubscribe"}:
+            return EmailSettingsSecurityPolicy(
+                request.find_service(EmailUnsubscribeService)
```

Comment:
Ah I reckon the change of including the full request here is to fetch the service.